src/ice_match/rule_0/main.py

```
@@ -19,133 +19,133 @@
-            
+
             # Show detailed trade breakdown if requested
             if show_details:
                 self.display.show_position_details(trader_matrix, exchange_matrix)
-            
+
         except FileNotFoundError as e:
             console.print(f"[red]Error: File not found - {e}[/red]")
             sys.exit(1)
         except Exception as e:
             console.print(f"[red]Error: {str(e)}[/red]")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)